tests/testsuite.src/run_misc.at

```diff
@@ -15131,3 +15131,94 @@ AT_CHECK([COB_PROF_ENABLE=1 COB_PROF_FILE=prof.csv $COBCRUN_DIRECT ./caller], [0
 ])
 
 AT_CLEANUP
+
+
+AT_SETUP([On-the-fly BCD normalization on MOVE from ALPHANUMERIC])
+
+AT_DATA([prog.cob], [
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. prog.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 SRC-ALNUM     PIC X(10)  VALUE "ABC456789}".
+       01 DST-NUMDISP   PIC S9(10).
+       01 DST-NUMEDIT   PIC +9B999B999B999.
+       01 DST-PACK      PIC S9(10) PACKED-DECIMAL.
+       01 DST-BIN       PIC S9(10) BINARY.
+       01 DST-BINC                 BINARY-CHAR SIGNED.
+       01 DST-BINS                 BINARY-SHORT SIGNED.
+       01 DST-BINL                 BINARY-LONG SIGNED.
+       01 DST-BIND                 BINARY-DOUBLE SIGNED.
+       01 DST-COMP5     PIC S9(10) COMP-5.
+       01 DST-COMP6     PIC  9(10) COMP-6.
+       01 DST-COMPX     PIC 9(10)  COMP-X.
+       01 DST-COMPN     PIC 9(10)  COMP-N.
+       01 DST-FLTSHORT             FLOAT-SHORT.
+       01 DST-FLTLONG              FLOAT-LONG.
+       01 DST-INDEX                INDEX.
+
+       PROCEDURE DIVISION.
+       MAIN.
+
+           DISPLAY "SRC-ALNUM        : '" SRC-ALNUM "'".
+
```

Comment:
`IF SRC-ALNUM NOT NUMERIC DISPLAY "NOT NUMERIC".` should be taken (I don't question it does but please add it to the test to ensure it does).